app/src/lib/stores/base-store.ts

```diff
@@ -0,0 +1,55 @@
+import { Emitter, Disposable } from 'event-kit'
+
+export abstract class BaseStore {
+  protected readonly _emitter = new Emitter()
```

Comment:
Because this has the right access modifier, let's rename this to `emitter` rather than spread the the `_` habit around.